string-manipulation-1/src/capitalize.js

```diff
@@ -0,0 +1,23 @@
+/* exported capitalize */
+// function capitalize(word) {
+//   var totalStrin = [];
+//   for (var i = 0; i < word.length; i++) {
+//     totalStrin.push((word[i].toLowerCase()));
+//   }
+//   // console.log(totalStrin);
+//   return totalStrin.toUpperCase();
+
+// }
```

Comment:
Please remove all commented code before turning in. You have some in some other files too.